cmd/counters/src/lib.rs

```diff
@@ -274,7 +278,9 @@ enum Subcmd {
         /// * "csv": outputs counters in comma-separated values (CSV) format,
         ///   suitable for use with other tools.
         ///
-        /// [conflicts with: --ipc]
+        /// * "json": outputs counters in JSON format, suitable for use with
+        ///   other tools.
+
```

Comment:
Accidental newline, or should we keep the `[conflicts with: --ipc]` note?
```suggestion
```

Reply:
the `[conflicts with: --ipc]` was removed as it incorrectly suggests that `--ipc` is a flag, rather than a subcommand. Since `ipc` is now a subcommand, Clap's help text for it won't suggest that you can pass `--output`  with it, so the "conflicts with" isn't needed any longer.